config/routes.rb

```diff
@@ -2,8 +2,10 @@
   namespace :api do
     namespace :v1 do
       resources :skills, only: [:index, :create, :show, :update, :destroy], defaults: { format: :json }
-      resources :groups, only: [:index], defaults: { format: :json }
+      resources :sections, only: [:index], defaults: { format: :json }
       resources :sessions, only: [:create]
     end
   end
+
+  get '/auth/:provider/callback', to: 'sessions#create', as: 'login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.